README.md

```diff
@@ -2,6 +2,13 @@
 
 Contains public Helm charts for various Lightbend projects. This project is hosted on [GitHub Pages](https://lightbend.github.io/helm-charts/index.yaml).
 
+
+## How to cut a new release
+```
+change version in enterprise-suite/Chart.yaml
```

Comment:
Turns out you don't need to do this anymore.  The script at scripts/make-release.sh will take care of this automatically.  But no point doing another release for that.